rockspecs/luaexpat-1.5.2-1.rockspec

```diff
@@ -0,0 +1,51 @@
+local package_name = "luaexpat"
```

Comment:
Please don't include new rockspecs or version bumps in this PR. A release may be warranted but the release process is actually going to be harder to manage with this in an unrelated PR than it would have been otherwise. If there are changes that need to be made to the rockspec to go along with the PR make it on the dev/scm rockspec instead. We'll propagate that on the next release.